script.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bash
+trap 'jobs -p | xargs -r kill' EXIT
+echo 'Running: '\''cd examples/demo-rollup/'\'''
+cd examples/demo-rollup/
```

Comment:
Nice! Looks like this it would be easy to get the `echo` and the actual command out of sync - maybe we should define a bash function which runs `echo` followed by `sh`?

Reply:
Oh, I see this is auto-generated. Nevermind! Did you intend to check it in?

Reply:
Yeah, but I'm removing it before merging. I committed it to facilitate reviews